docs/services/slam/cartographer/_index.md

```diff
@@ -254,6 +213,76 @@ Adjust these parameters to fine-tune the algorithm `cartographer` utilizes in as
 
 For more information, see the Cartographer [algorithm walkthrough](https://google-cartographer-ros.readthedocs.io/en/latest/algo_walkthrough.html), [tuning overview](https://google-cartographer-ros.readthedocs.io/en/latest/tuning.html), and [config parameter list](https://google-cartographer.readthedocs.io/en/latest/configuration.html).
 
+## Create or update a map
+
+### Online
+
+On the **Control** tab, you can start a mapping session.
+
+When you start a mapping session:
+
+- Viam spins up a copy of cartographer-module in the cloud to execute the Cartographer algorithm.
+  Because Cartographer's algorithm is CPU-intensive especially for creating or updating a map, in this mode the cartographer-module on your robot acts as a stub.
+- Viam's [Data Capture](/services/data/configure-data-capture/) continuously monitors and captures your robot's sensor data while the robot is running.
+- Cartographer uses the data captured from when you click **Start session** until you click **End session** to create the map.
+
+Once you click **End session**, the map is uploaded to the cloud(?) and visible on your Location page under **SLAM Library**.
```

Comment:
```suggestion
Once you click **End session**, the map is uploaded to the cloud and visible on your Location page under **SLAM Library**.
```

Reply:
haha that's here because I need the SME to review, it seems weird to me that the map is being produced in the cloud and then it's uploaded?